code/modules/admin/verbs/request_music.dm

```diff
@@ -0,0 +1,30 @@
+/mob/verb/request_music()
+	set category = "OOC"
+	set name = "Request Music"
+
+	if(GLOB.say_disabled) //This is here to try to identify lag problems
+		to_chat(usr, span_danger("Speech is currently admin-disabled."), confidential = TRUE)
+		return
+
+	var/msg = tgui_input_text(usr.client, "Please Input a URL", "Music Request", "")
+	//msg = copytext_char(sanitize(msg), 1, MAX_MESSAGE_LEN)
+	if(!msg)
+		return
+	log_prayer("[src.key]/([src.name]): [msg]")
+	if(usr.client)
+		if(usr.client.prefs.muted & MUTE_PRAY)
+			to_chat(usr, span_danger("You cannot pray (muted)."), confidential = TRUE)
+			return
+		if(src.client.handle_spam_prevention(msg,MUTE_PRAY))
+			return
+
+	var/msg_tmp = msg
+	msg = span_adminnotice("<b><font color='cyan'>MUSIC REQUEST: </font>[ADMIN_FULLMONTY(src)] [ADMIN_SC(src)]:</b> [span_linkify(msg_tmp)] [ADMIN_PLAY_MUSIC(msg_tmp)]")
+	for(var/client/C in GLOB.admins)
+		if(C.prefs.chat_toggles & CHAT_PRAYER)
+			to_chat(C, msg, type = MESSAGE_TYPE_PRAYER, confidential = TRUE)
+	to_chat(usr, span_info("You request music: \"[msg_tmp]\""), confidential = TRUE)
+
+	GLOB.requests.music_request(usr.client, msg_tmp)
+
+	SSblackbox.record_feedback("tally", "admin_verb", 1, "music_request") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```

Comment:
this isn't an admin_verb, why are you logging it in this table?

Reply:
What table should it be logged to if any?

Reply:
Do we log the frequency of "pray" verbs? If so, change it to that table. If not, just scrap this line.